consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/blockcreation/BftMiningCoordinator.java

```diff
@@ -115,6 +115,7 @@ public void stop() {
         LOG.debug("Interrupted while waiting for BftProcessor to stop.", e);
         Thread.currentThread().interrupt();
       }
+      eventHandler.reset();
```

Comment:
Reset seems odd here in that we trying to stop all processing of BFT here. A stop method to prevent any more processing in the controller would make sense. With everything stopped, we won't be processing BFT events anyway so maybe it's not worth it?

And by a stop method on the controller I'm thinking it would change the started state to false as it does right now in the reset method and also replace the block height manager with the noop version so it wouldn't do any processing of events.

Reply:
Yeah I originally called it `stop()`, but as it wasn't doing anything other than reset the state var I wasn't sure it was quite right. But I'm fine with using that term, and I'll add in replacing the block height manager at the same time as you've suggested.